docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md

```diff
@@ -0,0 +1,416 @@
+---
+
+プラグイン設定画面を開いたとき、プラグインの設定値を取得し、
+前回保存したときの値があればそれらを表示するようにします。
+
+```javascript
+// Display the value when saved last time
+const config = kintone.plugin.app.getConfig(PLUGIN_ID);
+if (Object.keys(config).length) {
+  datePicker.value = config.date;
+  dropdown.value = config.condition;
+  multiChoice.value = JSON.parse(config.targetFields);
+}
+```
+
+---
+### コンポーネントの配置
+---
+
+コンポーネントを配置する場所の要素を config.html 内の id を使って取得し、
+そこに各コンポーネントの要素を追加していきます。
+
+```javascript
+// Display components
+const dateSpaceEl = document.getElementById("date_space");
+dateSpaceEl.appendChild(datePicker);
+const dropdownSpaceEl = document.getElementById("dropdown_space");
+dropdownSpaceEl.appendChild(dropdown);
+const multiChoiceSpaceEl = document.getElementById("multichoice_space");
+multiChoiceSpaceEl.appendChild(multiChoice);
+const buttonSpaceEl = document.getElementById("button_space");
+buttonSpaceEl.appendChild(cancelButton);
+buttonSpaceEl.appendChild(saveButton);
+```
+
+---
+### ボタンクリック時の挙動
+---
+
+保存ボタンを押した時に、必須項目のチェックをしています。<br/>
+ひとつでも未入力の項目があるとエラーラベルを表示します。<br/>
+設定項目に問題がない場合は、確認ダイアログを表示します。
+
+```javascript
+// When the Save button is clicked
+saveButton.addEventListener("click", event => {
+  // Reset error messages
+  datePicker.error = "";
+  dropdown.error = "";
+  multiChoice.error = "";
+  // Check required itmes
+  let requiredFlag = false;
+  if (!datePicker.value) {
+    datePicker.error = "Please enter";
+    requiredFlag = true;
+  }
+  if (dropdown.value === "-----") {
+    dropdown.error = "Please select";
+    requiredFlag = true;
+  }
+  if (!multiChoice.value.length) {
+    multiChoice.error = "Please select";
+    requiredFlag = true;
+  }
+  if (requiredFlag) return;
+  dialog.open();
+  divEl.appendChild(dialogCancelButton);
+  divEl.appendChild(dialogOKButton);
+});
+// When the Cancel button is clicked
+cancelButton.addEventListener("click", event => {
+  history.back();
+});
+```
+
+---
+### ダイアログ内のボタンクリック時の挙動
+---
+
+意図しない設定の保存を避けるため、保存ボタンクリックの後に確認ダイアログを挟みます。<br/>
+
+![Confirmation](/img/dialog.png)
+
+OK ボタンを押すと、プラグインに設定値として持たせたい情報をオブジェクトに格納し、プラグインに保存します。
+
+```javascript
+// When the OK button in Dialog is clicked
+dialogOKButton.addEventListener("click", event => {
+  const selectedFields = JSON.stringify(multiChoice.value);
+  const date = datePicker.value;
+  const condition = dropdown.value;
+  const config = {
+    date: date,
+    condition: condition,
+    targetFields: selectedFields
+  };
+  kintone.plugin.app.setConfig(config);
+});
+// When the Cancel button in Dialog is clicked
+dialogCancelButton.addEventListener("click", event => {
+  dialog.close();
+});
+```
+
+## JavaScript/CSS カスタマイズ（アプリ画面）
+
+アプリ画面の実装コードを解説します。<br/>
+以下のようなJavaScript ファイル（desktop.js）を js フォルダに配置します。<br/>
+プラグイン設定画面のコンポーネントから値を引き継いで、アプリのカスタマイズで利用しているのがポイントです。<br/>
+DatePicker コンポーネントの value が yyyy-mm-dd 形式のため、
+比較しやすいように本日の日付も yyyy-mm-dd 形式で取得しています。<br/>
+今回は luxon.js という外部の日付ライブラリを利用しています。
+
+
+```javascript title="desktop.js"
```

Comment:
You changed "\*\*\*desktop.js***" to "title="desktop.js"".
It's different from v1.11.0.
Please check it.